lib/screens/register/register_screen.dart

```diff
@@ -71,16 +77,34 @@ class _RegisterFormState extends State<RegisterForm> {
     return null;
   }
 
+  String _validateUserName(String value) {
+    if (value.length < 4) {
+      return "Username cannot be less than 4 characters";
+    } else if (value.length > 26) {
+      return "Username cannot be more 26 characters";
+    }
+    return null;
+  }
+
   String _validateEmail(String value) {
-    if (value.isEmpty) {
-      return "Email cannot be empty"; // TODO: Add regex based validation
+    if (!EmailValidator.validate(value)) {
+      return "Email is not valid";
     }
     return null;
   }
 
   String _validatePassword(String value) {
-    if (value.isEmpty) {
-      return "Password cannot be empty"; // TODO: Add regex based validation
+    if (value.length < 8) {
+      return "Password should be longer than 8 characters"; // TODO: Add regex based validation
+    } else if (value.length > 64) {
+      return "Password should be no longer than 64 characters"; // TODO: Add regex based validation
+    }
+    return null;
+  }
+
+  String _validateConfirmPassword(String value) {
+    if (value.toString() != _passwordController.text) {
```

Comment:
@KevalPrajapati Can you remove tge #153 parts ? Lets keep it open for first timers for now

Reply:
Done.